### ISSUE TITLE
learning rate and other hyper-parameters

### ISSUE THREAD

**Issue by coolbay** [sunnyxiaohu/R-C3D.pytorch#41]
What learning rates are you using to train the model? I use the default hyper-parameters specified in the provided code and 4 GPUs to train the C3D model on THUMOS14. But I got a loss of NaN at [session 1][epoch  1][iter  301/3897].  I guess the hyper-parameters are not right. Could you please provide the hyper-parameters you used for training?

Thank you very much!
